plugins/env/app/models/environment_variable.rb

```diff
@@ -56,7 +56,10 @@ def resolve_secrets(project, deploy_group, env, preview:)
           found = resolver.read(secret_key)
```

Comment:
we don't need the `value` ... so can make a new `expand_key` method that returns the full path if the key is found ... then show the full path

Reply:
i opted for seeing how far i could get with existing API’s, but yep, that’d be preferable..

Should that be another PR? otherwise i can add it tomorrow to this one.

Reply:
.. unless you have time to provide something